.travis.yml

```diff
@@ -64,6 +64,7 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[all]"
+  - "./examples/app-recreate.sh"
```

Comment:
Hi Tibor, 
I read now the discussion. :smile: 
I agree with you when you say:
> One advantage of using shell script for this kind of "integration" testing is that it emulates precisely the commands we tell the user to do in the example app documentation. 

And agree with Lars when he says:
> I think we need a general solution which works for all our packages

That's my propose: at least for the installation instructions (as you wrote here in a single  is better to leave them inside a script and write addition tests in the pythonic way.
Because:
* The installation is a process "on/off". It means that it's enough that a single part fails to fails the entire process (and thats why I added many `|| exit 1` ). I see that you also added them in a [single step](https://github.com/inveniosoftware/invenio-records/blob/master/tests/test_example_app.py#L51-L54).
* Users can appreciate to have a unified and easy way to install the examples by run a single scripts. 
* If the requirements of an example change for any reason, you don't need to update tests.
* Part of tests became the same for all modules. It means save time writing them and save time update them.

WDYT @tiborsimko ?

In any case, I'm the idea of we need to test that the example simply works. 
I think we should not test any single aspect here because it should be already done in "normal" unit tests and in integration tests.